src/components/map/Map.scss

```
@@ -0,0 +1,24 @@
+// *{
+//     border: 2px solid red;
+// }
+
+main{
+   
+    position:relative;
+}
+
+.map img{
+    padding: 5%;
+    width:90%;
+    height: auto;
+}
+
+#Curiosity-day-1{
```

Prefer classes to ids:
```suggestion
.curiosity-day-1{
```
You'll need to change the corresponding bit in the other file too!

I think there's opportunity to commonise other classes as well - you could have a `.route-marker` class that sets `position: absolute` for all markers, so it doesn't have to be replicated here!